output_2017-10-26_2017-11-02.csv

```diff
@@ -0,0 +1,65 @@
+Search Term,Searched Repo,Searched From,Total Searches,Total Clicks,CTR,Mean Ordinality
```

Comment:
Did you mean to add this?

Reply:
Uff no - I absolutely didn't this has been fixed in master cc52267fa6d3877a38632e73f8f1d54e6c982d57.